scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -133,16 +118,15 @@ workflow GvsCalculatePrecisionAndSensitivity {
   }
 }
 
-task IsVcfOnChromosome {
+task IsVcfOnChromosomes {
   input {
     File input_vcf
-    String chromosome
+    Array[String] chromosomes
   }
 
   String output_vcf_name = basename(input_vcf)
-
-  command {
-    set -e -o pipefail
+  command <<<
+    set -eou pipefail
```

Comment:
seeing something below that's making me wonder if this is having the intended effect...
```suggestion
    set -o errexit -o nounset -o pipefail
```